tests/unit/test_server_exceptions.py

```diff
@@ -15,14 +15,23 @@
 
 """Test the base exception for servers."""
 
+import pydantic
 import pytest
 
 from httpyexpect.models import HTTPExceptionBody
```

Comment:
We should be consistent with naming, so either change all HTTP to CamelCase or let HTTP be caps everywhere :-D

Reply:
You are right, now everything is consistent